fabric/src/main/resources/fabric.mod.json

```diff
@@ -29,6 +29,6 @@
         "fabric-api": "*",
         "minecraft": ">=1.19.2",
         "java": ">=17",
-        "voicechat": "*"
+        "voicechat": ">=1.19-2.4.7"
```

Comment:
We definitely want to have some error if we aren't on a new enough version with the APIs we need, but I'm worried this might caused issues on 1.20+, so maybe something on startup

Reply:
I added a system to the mod initialiser that checks only the `2.4.7` part. Probably not the greatest way to do it, but it works. Will be done in the next commit.